### ISSUE TITLE
Reshoot-R doesn’t run for me

### ISSUE THREAD

**Issue by rsn8887** [MiSTer-devel/Minimig-AGA_MiSTer#31]
The new Amiga AGA game Reshoot-R doesn’t load for me in Mister core. It crashes during loading before the title screen appears. It works on WinUAE but even there it requires changing the “CIA Overlay” setting otherwise it hangs after the title screen.

**Comment by apolkosnik** [comment]
I've tried the final version, and it worked.

**Comment by rsn8887** [comment]
I bought my version from the official store, so I think I should also have the "final" version.

Maybe I used wrong settings? What settings did you use? Which core version did you try? I only tried the core from November 3rd, maybe it works in a different minimig core version?

EDIT: This is the game version that I am trying to run  https://www.amigashop.org/product_info.php?cPath=2_35&products_id=295
